python/pymemcache/requirements.txt

```diff
@@ -0,0 +1,2 @@
+-e /Users/kyle.verhoog/dev/dd-trace-py
```

Comment:
probably want this to just be `dd-trace-py`?

Reply:
I also noticed this is the same for the vertica example, but I don't see a PR for that, so commenting here :)